### ISSUE TITLE
Any chances you can keep a Changelog ?

### ISSUE THREAD

**Issue by luzpaz** [ColumPaget/Alaya#6]
Per https://keepachangelog.com/

It would help downstream package maintainers. TIA

**Comment by ColumPaget** [comment]
I'll try to develop the habit, it might also help with auto-announcement on freshcode, which I know uses changelogs for this. I kinda thought both tasks could be handled by git, but it's not working out that way.

**Comment by ColumPaget** [comment]
Well, I've managed to develop the habit, and have added changelogs for all my projects now!

**Comment by luzpaz** [comment]
Right on, I updated freshcode to reference the changelog now 👍 
https://freshcode.club/projects/alaya

**Comment by ColumPaget** [comment]
I must say, I keep trying to use changelogs at freshcode, and it keeps not working. Not sure if I'm doing something wrong there.

**Comment by luzpaz** [comment]
It works, you can test[^1] it, just click `Test-Run` button

[^1]: https://freshcode.club/drchangelog?autoupdate_module=name=&email=&link=http%3A%2F%2F&comment=&is_new=&name=alaya&title=Alaya&summary=Webdav+enabled+webserver+mostly+focused+on+file+storage&homepage=https%3A%2F%2Fgithub.com%2FColumPaget%2FAlaya&description=Alaya+is+a+chrooting+webserver+with+basic+webdav+extensions+and+a+number+of+unsual+features.+It+can+serve+both+http+and+https+and+is+intended+to+provide+a+simple+means+for+people+to+share+directories+with+webdav.+Although+it+chroots+it+supports+running+CGI+programs+outside+of+the+chroot+via+a+trusted-path+method.+Alaya+supports+perfect-forward-secrecy+and+client+certificate+authentication.&license=GNU+GPLv3&tags=storage%2C+webdav%2C+http%2C+server%2C+c%2C+posix%2C+linux&image=&version=4.6&state=stable&scope=minor+bugfix&changes=Bundled+libUseful+dir+now+named+libUseful-bundled+to+prevent+confusion+with+any+installed+libUseful.+Upgrade+to+libUseful-5%2C+this+should+with+rotating+logfiles.&download=https%3A%2F%2Fgithub.com%2FColumPaget%2FAlaya%2Farchive%2Fv%24version.tar.gz&urls=Github+%3D+https%3A%2F%2Fgithub.com%2FColumPaget%2FAlaya%0D%0AChangelog+%3D+https%3A%2F%2Fgithub.com%2FColumPaget%2FAlaya%2Fblob%2Fmaster%2FCHANGELOG&autoupdate_module=changelog&autoupdate_url=https%3A%2F%2Fraw.githubusercontent.com%2FColumPaget%2FAlaya%2Frefs%2Fheads%2Fmaster%2FCHANGELOG&autoupdate_regex=&submitter=ColumPaget&lock=&.ct=769a470dd7a8de0223f6f5b83377666b75392595&import_via=JSON&import_name=

**Comment by ColumPaget** [comment]
Yeah, I've done that, but the projects never auto-update. For instance, 'getlock' is 2 releases ahead of freshmeat, but when I click on 'Test-Run' Dr Changelog says it can parse the changelog.

**Comment by luzpaz** [comment]
Both projects should auto-update from here on out.
